5.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <title>5</title>
+</head>
+<body bgcolor="#000" background="https://guardian.ng/wp-content/uploads/2022/07/World-population.jpg">
+    <style>
+        article {
+         background: #3e3e3e;
+         color: #000000;
+         padding: 10px;
+        }
+        
+    </style> 
+    <header>
+        <p><b><font size="72" color="white" face="Arial">The World</font></b></p>
```

Comment:
```suggestion
        <h1>The World</h1>
```
не нужны тут font, b